manifests/init.pp

```diff
@@ -90,6 +90,9 @@
 # $ca_key_password::              CA key password
 #                                 type:String
 #
+# $ciphers::                      Allowed ciphers for ssl connection. Array of strings
+#                                 type:Array
```

Comment:
You can use ```type:Array[String]``` to enforce that it's an array of strings in the installer. When we move to Puppet 4 we can also easily enforce it in the language itself.